multinet/downloaders/d3_json.py

```diff
@@ -25,52 +26,60 @@ def download(workspace: str, graph: str) -> Any:
     `graph` - the target graph
     """
 
-    nodes = []
-    links = []
-
+    table_nodes_pattern = re.compile(r"^([^\d_]\w+)_nodes(/.+)")
```

Comment:
Can you put in a comment explaining how this regex works? Besides that, it doesn't seem like we should assume the node tables have that `_nodes` suffix right?

Reply:
Yeah I can add a comment. This assumption mirrors the functionality from the `d3_json` uploader, but I agree we should also support the case where that suffix isn't present.

Reply:
Actually, this does support both cases currently. If the regex matches, it overwrites the `source` and `target` variables, but otherwise leaves the `source` and `target` variables (representing the source and target node tables) unchanged.